tests/xrefs.py

```diff
@@ -1,6 +1,6 @@
 """
 Blah.
-L{undefined_xref_from_module}
+`undefined_xref_from_module`
```

Comment:
These should be double back ticks, I believe.

Reply:
Double back ticks are for code sample - [**ref**](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup).
The expected replacement for cross-reference links ***L{text}*** is [***:py:obj:\`text\`***](http://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-syntax). We skip prepending the ***':py:obj:'*** as we specify [***default_role***](http://www.sphinx-doc.org/en/master/usage/configuration.html#confval-default_role) in config file (conf.py).

>***default_role='py:obj'***

**Minor**: We use [**py:obj**](http://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#role-py:obj), as we aren't sure of the type of the referenced item.